packages/bpk-mixins/src/mixins/_forms.scss

```diff
@@ -611,6 +650,10 @@
     }
   }
 
+  &:checked:not(:indeterminate) {
+    @include bpk-checkbox__checkmark;
+  }
+
```

Comment:
`:not(:indeterminate)` needed so that indeterminate still takes precedence over checked.